lib/routes/gdufs/news.ts

```diff
@@ -0,0 +1,89 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+
+const site = 'https://www.gdufs.edu.cn';
+
+export const route: Route = {
+    path: '/news',
+    categories: ['university'],
+    example: '/gdufs/news',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['https://www.gdufs.edu.cn/gwxw/gwxw1.htm', 'https://www.gdufs.edu.cn/'],
+        },
+    ],
+    name: '广东外语外贸大学-新闻',
```

Comment:
Do not repeat namespace name https://github.com/DIYgod/RSSHub/blob/1f9bfc5f7e5d1f0bfd03d7565aeb90e91cbc41a5/lib/routes/gdufs/namespace.ts#L4 in route name
```suggestion
    name: '新闻',
```